### PR TITLE
feat(file-picker): add `limit` option for iOS

### LINKED ISSUE #17: feat(file-picker): Create the option to define a limit for the number of files picked by user
**Is your feature request related to a problem? Please describe:**

I would like if can define a max number of files picked by user, for delimite (how like whatsapp). This would prevent the user from choosing infinite files.

**Describe the solution you'd like:**

Just an argumento in options the file picker for limit this.
like: limit?: 10 (the user can upload 10 files).

**Describe alternatives you've considered:**

**Additional context:**

### PR DESCRIPTION
**Pull request by stephan-fischer**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] The changes have been tested successfully.
- [x] A changeset has been created (`npm run changeset`).
- [x] I have read and followed the [pull request guidelines](https://capawesome.io/contributing/pull-requests/).

Close #17

### CODE REVIEW

**Comment by danbunkr** [comment]
@robingenz - This would be a huge win for our application if we could get this merged even if its just for IOS

File: .changeset/heavy-dancers-impress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-file-picker': patch
+---
+
+feat(file-picker): Create the option to define a limit for the number of files picked by user - only on ios
```
Comment (robingenz) [inline .changeset/heavy-dancers-impress.md]:
```suggestion
feat(ios): add `limit` option
```

File: .changeset/heavy-dancers-impress.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-file-picker': patch
```
Comment (robingenz) [inline .changeset/heavy-dancers-impress.md]:
```suggestion
'@capawesome/capacitor-file-picker': minor
```

**Review by robingenz — CHANGES_REQUESTED** [review]
Please also update the `definitions.ts` file and run `npm run docgen`:

```diff
export interface PickMediaOptions {
+  /**
+   * The maximum number of files that the user can select.
+   *
+   * This option is ignored if `multiple` is set to `false`.
+   *
+   * Only available on iOS.
+   *
+   * @example 4
+   */
+  limit?: number;
  ...
```

**Review by stephan-fischer — COMMENTED** [review]
thank you!

**Comment by stephan-fischer** [comment]
@robingenz do i have to accept somewhere the changes? Sorry first time here creating a pull request and accepting changes

**Comment by robingenz** [comment]
@stephan-fischer No problem, that will be squashed anyway.

**Review by robingenz — APPROVED** [review]

**Comment by robingenz** [comment]
@stephan-fischer Please run `npm run fmt`.

**Review by robingenz — CHANGES_REQUESTED** [review]
Please run `npm run fmt`.

**Review by stephan-fischer — COMMENTED** [review]
Reviewed

**Review by robingenz — APPROVED** [review]
Thank you!

**Pull request merged**